modules/statics/src/coins.ts

```diff
@@ -4265,6 +4273,22 @@ export const coins = CoinMap.fromCoins([
     '0x7968bc6a03017ea2de509aaa816f163db0f35148',
     UnderlyingAsset.HGET
   ),
+  erc20(
+    '50bc024c-f6e2-4b69-8010-be20243a1c35',
+    'high',
+    'Highstreet',
+    6,
```

Comment:
this should be 18 according to: https://etherscan.io/token/0x71Ab77b7dbB4fa7e017BC15090b2163221420282